nasa-gibs/worldview
# WV-2886 Tour Story Animation Caching

## Description
This removes the clearing of the cache between steps of a tour story, to eliminate the need for a long loading time for an animation that has already played (and therefore has been cached).

## How To Test
1. `git checkout wv-2886-tourstory-animationcache`
2. `npm ci`
3. `npm run watch`
4. Open any Tour Story with an animation (for example, 'Geostationary Imagery Every 10 Minutes!').
5. Proceed to the step with an animation and click the play button. Let it determine the buffer size and preload the buffer, and let the animation play out.
6. After the animation is played, go forward a step in the tour story, and then go back again to the previous step and press the play button again to play the animation.
7. Verify that the site doesn't need to determine the buffer size and preload the buffer, and that the animation plays immediately after clicking the play button. Verify that the animation looks correct, too.
8. Without reloading the page, finish the tour story, open a different tour story and go through that one, too. Then open the original tour story, go to the step with the animation, play the animation and verify that it did not need to load and that it looks correct.
